system/index.js

```diff
@@ -144,8 +145,9 @@ System.prototype.hasLegalTerms = function() {
  * @return {boolean}
  */
 System.prototype.hasLegalPolicy = function() {
-    try {
-        return this.getConfig(['legal', 'policy']) !== false;
+	let config = this.getConfig(['legal', 'policy']);
+	try {
```

Comment:
this try block is unnecessary in it's current implementation, as this does most likely not throw any error. However, getConfig() can throw an error, if the configuration path (legal->policy) does not exist. In this case, the try block prevented the error to stop the application and return a reasonable default value. This is now not the case anymore.